src/main/java/edu/kis/powp/jobs2d/command/json/JsonCommandImporter.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.powp.jobs2d.command.json;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.stream.JsonReader;
+import edu.kis.powp.jobs2d.command.CompoundCommand;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class JsonCommandImporter {
+	private static final Gson gson = new GsonBuilder()
+			.registerTypeAdapter(DriverCommand.class,new JsonCommandSerializer())
+			.setPrettyPrinting()
+			.create();
+
+	public static CompoundCommand importCommand(String filepath) throws IOException {
+		JsonReader reader = new JsonReader(new FileReader(filepath));
+		return gson.fromJson(reader, CompoundCommand.class);
+	}
```

Comment:
importCommand shouldn't work with files (using simple string would make solution more open).
Moreover, there might be a need to support other input/output formats and this design doesn't make it easy to add new format.